src/cascadia/TerminalSettingsModel/ActionMap.cpp

```diff
@@ -15,36 +15,42 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 {
     static InternalActionID Hash(const Model::ActionAndArgs& actionAndArgs)
     {
-        size_t hashedAction{ HashUtils::HashProperty(actionAndArgs.Action()) };
+        static constexpr auto initialHash = til::hasher{}.finalize();
+        const auto action = actionAndArgs.Action();
+        til::hasher hasher;

```

Comment:
i'm confuse! this doesn't mix in the action, so two argument-less actions will hash to the same thing ... right?

Reply:
The `action` is being written into the hasher as the last thing in this function.
My code clarity here is clearly not quite optimal. 😅